client/my-sites/sidebar/sidebar.jsx

```diff
@@ -106,6 +106,10 @@ module.exports = React.createClass( {
 	stats: function() {
 		var site = this.getSelectedSite();
 
+		if ( ! site.capabilities ) {
```

Comment:
To check my understanding, we add all of these conditionals because we don't have a full site object when adding via invite?